README.md

```diff
@@ -1,14 +1,13 @@
 # HTTP 267 Doubtful But Okay
-Предлагается создать HTTP-статус имени Олега Тинькова `267 Doubtful But Okay`.
+It is proposed to create an HTTP status for the name Oleg Tinkov `267 Doubtful But Okay`.
+This web server or application response status means that the parameters (URL or data) of the request have questionable semantics (perhaps conflicting data was passed by the client), but the request is nevertheless accepted and processed.
```

Comment:
Because you dropped a blank line between these two lines, it will be rendered as if they are one paragraph (unlike the previous README, where these are two separate paragraphs)

Reply:
Thank you, just updated it so it will  look like an original